hooks/01_rpi-firmware.yaml

```diff
@@ -0,0 +1,6 @@
+name: "Raspberry Pi post disk hook"
+stages:
+    after-disk:
+    - name: "Copy firmware to EFI partition"
```

Comment:
Isn't this going to be executed in all disk builds? Raspberry and non raspberry, aarch64 and x86. If so probably we could think of some sort of if clause to limit the scope of this hook.